moped-database/migrations/1646261755800_alter_table_public_moped_project_drop_column_start_date/down.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE "public"."moped_project" ADD COLUMN "start_date" date;
+ALTER TABLE "public"."moped_project" ALTER COLUMN "start_date" DROP NOT NULL;
```

Comment:
I think it was a just in case thing. If it's not necessary I can leave it off!

Reply:
ok cool. I trust it doesn't hurt anything if it deployed fine in test covid negative 👍

Reply:
Mateo, you're correct - this isn't needed but also isn't harmful. I just did some testing to confirm this, and the default behavior of my local moped postgres service is to create the column without any `NOT NULL` constraint. This makes that second line an effective non-op, and it can go for clarity, but if it stays in, no harm in it. Thanks for pointing this out!